frontend/src/libs/components/icon/icons/icon-map.tsx

```diff
@@ -0,0 +1,17 @@
+import { type ValueOf } from '#/libs/types/types.js';
+import { IconNameToIcon } from '#libs/enums/enums.js';
+
+import { ReactComponent as Arrow } from './arrow.svg';
+import { ReactComponent as Plus } from './plus.svg';
+import { ReactComponent as SignOut } from './sign-out.svg';
+
+const iconNameToPlainSvgMap: Record<
+  ValueOf<typeof IconNameToIcon>,
+  JSX.Element
+> = {
+  [IconNameToIcon.ARROW]: <Arrow />,
+  [IconNameToIcon.PLUS]: <Plus />,
+  [IconNameToIcon.SIGN_OUT]: <SignOut />,
```

Comment:
```suggestion
  [IconNameToIcon.ARROW]: ArrowIcon,
  [IconNameToIcon.PLUS]: PlusIcon,
  [IconNameToIcon.SIGN_OUT]: SignOutIcon,
```

Reply:
when i change my icon-map to this i get error
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92929504/1da0b297-b7cd-4331-8255-e01fb5998500)

Reply:
I also got a bit lost with this part

Reply:
Maybe it's: 

`  [IconNameToIcon.ARROW]: <ArrowIcon />,
  [IconNameToIcon.PLUS]: <PlusIcon />,
  [IconNameToIcon.SIGN_OUT]: <SignOutIcon />,`

Reply:
It's weird, but I don't have this console error after renaming.